gnomad_qc/v4/analyses/grpmax_comps.py

```diff
@@ -102,30 +107,16 @@ def version_stats(
     t_variants = ht.count()
     if can_only:
         logger.info("Filtering to only MANE Select and canonical transcripts")
-        ht = ht.explode(ht.vep.transcript_consequences)
-        if version == "v2":
-            ht = ht.filter(hl.is_defined(ht.vep.transcript_consequences.canonical))
-        else:
-            ht = ht.filter(
-                hl.is_defined(ht.vep.transcript_consequences.canonical)
-                | hl.is_defined(ht.vep.transcript_consequences.mane_select)
-            )
-        logger.info(
-            "Filtering on mane_select and canonical transcript consequence term to "
-            "keep only non-synonymous variants..."
-        )
+        ht = filter_vep_to_canonical_transcripts(ht, filter_empty_csq=True)
+        # All MANE select transcripts in v4 are also the canonical transcript
```

Comment:
minor comment that this comment makes a little more sense above the call to `filter_vep_to_canonical_transcripts`